interfaces/api/storage/IMapManager.h

```diff
@@ -109,6 +110,17 @@ class XPCF_IGNORE IMapManager :
 	/// @return FrameworkReturnCode::_SUCCESS if succeed, else FrameworkReturnCode::_ERROR_
     virtual FrameworkReturnCode removeKeyframe(const SRef<SolAR::datastructure::Keyframe> keyframe) = 0;
 
+    /// @brief Add a 2d mask to map manager
+    /// @param[in] mask 2D mask to be added to map manager
+    /// @param[in] defineId if true an id will be set for the added mask, if false the id of the input mask will be used
+    /// @return FrameworkReturnCode::_SUCCESS if succeed, else FrameworkReturnCode::_ERROR_
+    virtual FrameworkReturnCode addMask2D(SRef<SolAR::datastructure::Mask2D> mask, bool defineId = true) = 0;
```

Comment:
Use "const" for "mask" parameter:
` virtual FrameworkReturnCode addMask2D(const SRef<SolAR::datastructure::Mask2D> mask, bool defineId = true) = 0;`